lib/Parser/XML/Xcal.rng

```diff
@@ -0,0 +1,1192 @@
+# RELAX NG Schema for iCalendar in XML
```

Comment:
In Hoa, we always put all the resources inside the library, and we access it through `hoa://Library/<name>/<path>/`, for instance `hoa://Library/Ruler/Grammar.pp` to access to the grammar of `Hoa\Ruler`. This is very convenient.

But, according to the structure of sabre libraries, yup, maybe it's better to have a `resources/schema/` directory? I am going to create it.